src/components/mapComponents/constants.tsx

```diff
@@ -63,3 +63,22 @@ export const SITE_OPTIONS: { label: ReactNode; value: string }[] = [
     value: 'Open',
   },
 ];
+
+export const LIGHT_MAP_STYLES: google.maps.MapTypeStyle[] = [
```

Comment:
⛏️ Can you leave a comment here linking the google maps documentation we used and explaining what we're hiding here?